modules/swagger-codegen/src/main/java/io/swagger/codegen/AbstractGenerator.java

```diff
@@ -108,7 +108,7 @@ public String getFullTemplateFile(CodegenConfig config, String templateFile) {
         }
             
         // Fall back to the template file embedded/packaged in the JAR file...
-        return config.embeddedTemplateDir() + File.separator + templateFile;
```

Comment:
@shijinkui the reason why we use File.separator is to make sure it works with Windows, Linux and Mac.

We do have `bin/windows/akka-scala-petstore.bat` to cover `akka-scala` in the Windows platform.

Are you encountering the issue with 2.3.0 JAR or a local JAR built from the latest master?

Reply:
latest master: 3.0 snapshot

Reply:
I just couldn't the issue with missing file in the customized template folder: https://ci.appveyor.com/project/WilliamCheng/swagger-codegen-wh2wu/build/test_windows_template-7192

Branch: https://github.com/swagger-api/swagger-codegen/tree/test_windows_template

Reply:
We cannot merge this as this change in abstract generator will impact all generators

Reply:
@shijinkui I was not able to reproduce the issue. I'm on Mac. Also as @wing328 mentioned your solution is not platform independent.